op-chain-ops/interopgen/recipe.go

```diff
@@ -200,12 +204,15 @@ func InteropL2DevConfig(l1ChainID, l2ChainID uint64, addrs devkeys.Addresses) (*
 				BaseFeeVaultRecipient:                    baseFeeVaultRecipient,
 				L1FeeVaultRecipient:                      l1FeeVaultRecipient,
 				SequencerFeeVaultRecipient:               sequencerFeeVaultRecipient,
+				OperatorFeeVaultRecipient:                operatorFeeVaultRecipient,
```

Comment:
should be able to delete per https://github.com/ethereum-optimism/optimism/pull/12166/files#r1941779240